.github/ISSUE_TEMPLATE/deployment.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Deployment
+about: Use this template when you're going to deploy a new release of the extension and web app.
+title: Publish vX.X.X extension + web app
+labels: deployment
+---
+
+- [ ] Manual testing to confirm extension works with main flows
+  - [ ] Balances
+  - [ ] Send
+  - [ ] Swap
+  - [ ] Staking
+  - [ ] IBC
+  - [ ] Tx details
+  - [x] ~Do a full sync with with [assertRootValid()](https://github.com/penumbra-zone/web/blob/main/packages/query/src/block-processor.ts#L383-L395) enabled. Will throw if TCT roots are not correct.~ (Ignore for now.)
+- [ ] If there is already a changelist PR (like [this one](https://github.com/penumbra-zone/web/pull/799)), merge it. Otherwise, manually run `pnpm changeset version`.
+  - This step will publish updated npm packages.
+- [ ] Update [npm package version](https://github.com/penumbra-zone/web/blob/main/package.json#L3)
+- [ ] Update [manifest version](https://github.com/penumbra-zone/web/blob/main/apps/extension/public/manifest.json#L4) based on the extension's newly updated [`package.json` version](https://github.com/penumbra-zone/web/blob/main/apps/extension/package.json) in the extension.
```

Comment:
@grod220 these require being committed to a PR branch, then reviewed and merged, right?

Reply:
OK. Outside the scope of this PR, but we should discuss deployment workflows soon. If the person running a deployment has to make two separate PRs (one for this, and one for changesets above), AND wait for approval on them, deployments will be a huge pain.